### ISSUE TITLE
Getting a "MODULE_NOT_FOUND" on fresh install

### ISSUE THREAD

**Issue by marwinhormiz** [t3-oss/create-t3-app#218]
I created a new T3 App with only Tailwind and tRPC and when I start the dev server I get a module not found / require stack of a bunch of files. Don't really know what the issue here is but is seems like some files are missing that I did not get with create-t3-app, or maybe I have done something wrong? I tried the install 3 times and I get the same error everytime. 

![4c666054fd7ae3cc08b0badaead5c5af](https://user-images.githubusercontent.com/56167333/179604721-48b272f8-9d5a-4ce0-8b20-14cba93aa41c.png)

**Comment by juliusmarminge** [comment]
What package manager did you use when bootstrapping and also what Node version are you on?

**Comment by marwinhormiz** [comment]
I used npm and I am on v14.8.0

**Comment by juliusmarminge** [comment]
Interesting. I can't even get the CLI to run when using Node 14. Would you mind testing with Node 16?

<img width="898" alt="CleanShot 2022-07-18 at 22 21 53@2x" src="https://user-images.githubusercontent.com/51714798/179610509-635ea089-0b92-4117-adb8-c0c1874ecfa1.png">

**Comment by marwinhormiz** [comment]
So I found the issue and it's resolved now. The path to "node_modules/use-sync-external-store/shim/index.js" from useSyncExternalStore.js in the react-query source file (path shown in image at top) had the wrong path by original. The path in the commented line next to it is what is was from original, and I updated it as shown in the image and it works. Don't know if this is an issue from react-query in this case since it's their files the issue is at.
 
![74b9ce3027e5d45074296970ea777439](https://user-images.githubusercontent.com/56167333/179613651-3ecdc398-df89-4a4d-979e-cf99b8a28ba3.png)
